docs/content/docs/user-docs/cross-region-resource-management.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Cross-Region Resource Management"
+description: "Managing resources across different AWS regions"
+lead: ""
+draft: false
+menu: 
+  docs:
+    parent: "installing"
+weight: 50
+toc: true
+---
+
+If you are using resources across different regions, you can override the default region of a given ACK service controller. ACK service controllers will first look for a region in the following order:
+
+  1. The region annotation `services.k8s.aws/region` on the resource. If provided, this will override the namespace default region annotation.
+  2. The namespace default region annotation `services.k8s.aws/default-region`.
+  3. Controller flags, such as the `aws.region` variable in a given Helm chart
```

Comment:
Perhaps clarify with:

"The `--aws-region` controller flag. This flag may be set using the `aws.region` Helm chart variable."